ksql-execution/src/main/java/io/confluent/ksql/execution/ddl/commands/CreateStreamCommand.java

```diff
@@ -26,22 +27,22 @@
 
 @Immutable
 public class CreateStreamCommand extends CreateSourceCommand {
-
   public CreateStreamCommand(
-      final String sqlExpression,
-      final SourceName sourceName,
-      final LogicalSchema schema,
-      final Optional<ColumnName> keyField,
-      final TimestampExtractionPolicy timestampExtractionPolicy,
-      final Set<SerdeOption> serdeOptions,
-      final KsqlTopic ksqlTopic
+      @JsonProperty(value = "sqlExpression", required = true) final String sqlExpression,
+      @JsonProperty(value = "sourceName", required = true) final SourceName sourceName,
+      @JsonProperty(value = "schema", required = true) final LogicalSchema schema,
+      @JsonProperty(value = "keyField", required = true) final Optional<ColumnName> keyField,
+      @JsonProperty(value = "timestampExtractionPolicy", required = true)
```

Comment:
```suggestion
      @JsonProperty(value = "timestampExtractor", required = true)
```

Let's keep the names shorter. ;)

Reply:
This field is going away shortly anyway.